index.html

```diff
@@ -28,6 +26,138 @@ <h3 class="job-title">I'm a Software Developer</h3>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+                <div class="project5 project-sample">
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+                <div class="project6 project-sample">
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="full-btn">See Project</button>
+                </div>
+                <div class="project7 project-sample">
+                    <div class="content-wrapper">
+                        <h3 class="project-name">Profesional Art Printing Data</h3>
+                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="project-lang">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    </section>
+                    <button type="button" class="full-btn">See Project</button>
+                    <section class="about-me-section">
+                        <div class="grid-wrapper">
+                            <div class="aboutme">
+                                <h2 class="title">About me</h2>
+                                <p class="summary">
+                                    Hello I'm a software developer! I can help you build a product , feature or website Look through 
+                                    some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                                </p>
```

Comment:
- [ ] Kindly make sure you stuck to the design as much as possible (e.g., `font`, colors, images, text, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 
- [ ] for good semantic tags you may us `h1` tag imside of using `h2` 👍. Kindly fix it 😃.   
> h1 : font-family: 'Crete Round';

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![about](https://user-images.githubusercontent.com/57408419/227959450-04f49c24-55f7-43f1-9b97-f47da550f50e.JPG)| |![abouterror](https://user-images.githubusercontent.com/57408419/227959838-383ee0db-1c9a-47a7-ab3a-7fd232fbdd17.JPG)| |